TemporaryProjects.15.0/StartPageExtenderPackage.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading;
+using System.Runtime.InteropServices;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Shell;
+using Task = System.Threading.Tasks.Task;
+
+namespace TemporaryProjects
+{
+    [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
+    [InstalledProductRegistration("#110", "#112", "1.0", IconResourceID = 400)] // Info on this package for Help/About
+    [Guid(PackageGuidString)]
+    [SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1650:ElementDocumentationMustBeSpelledCorrectly", Justification = "pkgdef, VS and vsixmanifest are valid VS terms")]
+    [ProvideAutoLoad(StartPageToolWindowString, PackageAutoLoadFlags.BackgroundLoad)]
```

Comment:
This will now only load when the `StartPage` tool window becomes visible, which is exactly what we want!

Reply:
Awesome, thanks! How did you find out about this?

Reply:
I asked a question on the `extendvs` Gitter and after no response, just tried a few things. 😆